### ISSUE TITLE
Wrong example in the documentation for period_indicator

### ISSUE THREAD

**Issue by jmsimonma-md** [sdmx-twg/vtl#458]
the following example(Example 2) from the documentation on Time operators >Period indicator `DS_r := DS_1 [ filter period_indicator ( Id_3 ) = “A" ]` I think it is not correct because you cannot implicitly cast between time period and string.

**Comment by linardian** [comment]
Cast between time period and string is implicit, so I think the example is correct. @jmsimonma-md ?

**Comment by jmsimonma-md** [comment]
Agree, you're right,  now in 2.1 version is implicit so Im going to close this issue  ( when I opened this issue the cast wasn't implicit but now yes).
